.rubocop.yml

```diff
@@ -41,6 +41,8 @@ Layout/SpaceBeforeSemicolon:
   Enabled: false
 Layout/SpaceInsideHashLiteralBraces:
   EnforcedStyle: space
+Lint/ConstantResolution:
+  Enabled: false
```

Comment:
neat idea, but way too noisy. I think there is an `Only: [...]` option that we could apply/add, if we ever run into issues that this cop could help to catch/prevent.